openreview/conference/matching.py

```diff
@@ -605,6 +613,26 @@ def _build_subject_area_scores(self, submissions):
             raise openreview.OpenReviewException('Failed during bulk post of edges! Scores found: {0}, Edges posted: {1}'.format(len(edges), edges_posted))
         return invitation
 
+    def _compute_scores(self, score_invitation_id, submissions, matching_status):
+        invitation = self._create_edge_invitation(score_invitation_id)
+
+        job_id = self.client.request_expertise(name=self.conference.get_short_name(), group_id=self.match_group.id, paper_invitation=self.conference.get_blind_submission_id(), model='specter+mfr')
+        status = False
+        call_count = 0
+        while not status:
+            call_count+=1
+            status = 'Completed' in self.client.get_expertise_status(job_id['job_id'])['results'][0]['status']
+            if call_count == 30:
+                break
+            time.sleep(30)
+        if status:
+            result = self.client.get_expertise_results(job_id['job_id'])
+            scores = [[entry['submission'],entry['user'],entry['score']] for entry in result['results']]
+            matching_status['no_profiles'] = result['metadata']['no_profile']
+            matching_status['no_publications'] = result['metadata']['no_publications']
+            return self._build_note_scores(invitation, scores, submissions), matching_status
+        return None, matching_status
```

Comment:
can you add a try/except in case there is an error with the API and we should post a comment with the error.